image-fetcher/Makefile

```diff
@@ -2,17 +2,25 @@
 
 PROJECT = $(shell gcloud config get-value project)
 
-build:
+FETCH_LATEST = gcr.io/$(PROJECT)/image-fetcher:latest
+FETCH_IMAGE = gcr.io/$(PROJECT)/image-fetcher:$(shell docker images -q --no-trunc image-fetcher | sed -e 's,[^:]*:,,')
+
+fetch-image.sh: fetch-image.sh.in
+	sed -e "s,@project@,$(PROJECT)," \
+	  < fetch-image.sh.in > fetch-image.sh
+
+build: fetch-image.sh
 	docker build . -t image-fetcher
 
-push: IMAGE = gcr.io/$(PROJECT)/image-fetcher:$(shell docker images -q --no-trunc image-fetcher | sed -e 's,[^:]*:,,')
+push: 
```

Comment:
Is this meant to be empty?

Reply:
Certainly not, good catch.  Fixed here: https://github.com/hail-is/hail/pull/5780